x-pack/plugins/timelines/public/hooks/use_add_to_case.ts

```diff
@@ -140,6 +159,7 @@ export const useAddToCase = ({
     }
   }, [onClose, closePopover, dispatch, eventId]);
   return {
+    caseAttachments,
     addNewCaseClick,
```

Comment:
@academo I was wondering if we can get rid of `addNewCaseClick`. This is currently used here https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/public/components/actions/timeline/cases/add_to_case_action_button.tsx#L49, which actually comes down to https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/public/plugin.ts#L100, but I don't see `getAddToCasePopover` being used anywhere.

My understanding is that `add_to_new_case_button.tsx` is currently in use, where as `add_to_case_action_button.tsx` is a leftover. If that's the case, could you get rid of unused stuff in the follow up PRs you are working on?

Reply:
Hi @mgiota that it is indeed part of the plan. Please check the epic about this chain of PRs. https://github.com/elastic/kibana/issues/123183